packages/image_picker/image_picker_ios/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 * Removes unnecessary imports.
 
+## 0.8.6
```

Comment:
Please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#updating-a-changelog-that-has-a-next